model/minsky.cc

```diff
@@ -713,6 +717,103 @@ namespace minsky
        });
     canvas.requestRedraw();
   }
+  
+  /*
+  For using GSL non-linear least square fitting routines
+  patterned on first Large Nonlinear Least Squares Example.
+  See online documentation at https://www.gnu.org/software/gsl/doc/html/nls.html
+  */
+      
+namespace
+{	
+	
+   struct model_params {
+	   double *t;
+	   double *x_init;
```

Comment:
I have added a comment to this effect, thanks. I did not manage to rewrite the structure using references instead of pointers, so it will also have to done when I rework the implementation.